src/client/datascience/jupyter/jupyterDebugger.ts

```diff
@@ -54,13 +53,18 @@ export class JupyterDebugger implements IJupyterDebugger, ICellHashListener {
         if (!notebook) {
             throw new Error('Notebook not initialized');
         }
-        const result = await this.installer.isProductVersionCompatible(
-            Product.ipykernel,
-            '>=6.0.0',
-            notebook.getKernelConnection()?.interpreter
-        );
+
```

Comment:
Maybe put this into a common function somewhere?

Reply:
It would make it easier to handle the case where the ipykernel version is greater than 10.

Reply:
yees, I thought this was urgent but it isn't.
So I'll add it here.

Reply:
I agree this should be some common code,

Reply:
You can leave this here and fix in debt week or fix this now, i leave that upto you.

Reply:
If not doing now, please file an issue so we don't forget (to fix in debt week)